stdlib/package.json

```diff
@@ -27,7 +27,7 @@
     "index.js"
   ],
   "scripts": {
-    "clean": "del-cli \"**/*.wasm\" \"**/*.wat\" \"**/*.modsig\""
+    "clean": "del-cli \"**/*.wasm\" \"**/*.wat\" \"**/*.gro\" \"**/*.modsig\""
```

Comment:
Should we leave the `*.gr.wasm` in here as well for a version or two so we make sure to clean up the legacy files?

Reply:
That's covered by `**/*.wasm`.